shared/oval/inventory/installed_OS_is_part_of_Unix_family.xml

```diff
@@ -4,6 +4,7 @@
       <title>Installed operating system is part of the Unix family</title>
       <affected family="unix">
         <platform>multi_platform_all</platform>
+        <product>Webmin</product>
```

Comment:
This is actually an incorrect chnage and should not be added. Either delete `<criterion comment="Installed operating system is part of the unix family" test_ref="test_unix_family" />` from `Webmin/templates/static/oval/*` or change `<criterion comment="Installed operating system is part of the unix family" test_ref="test_unix_family" />`to `<extend_definition comment="Installed OS is part of the Unix family" definition_ref="installed_OS_is_part_of_Unix_family" />`